removed

```diff
@@ -423,5 +423,11 @@
     "reason": "Archived",
     "removal_type": "remove",
     "link": "https://github.com/hacs/integration/issues/1809"
+  },
+  {
+    "repository": "edenhaus/ha-prosenic",
+    "reason": "Archived",
+    "removal_type": "remove",
+    "link": ""
```

Comment:
If you include the `link` attribute you might aswell add the link of this PR for better accountability lateron:
`https://github.com/hacs/default/pull/807`